TACC/tup-ui
# feat(tup-cms): svg snippets, more choices

## Overview

Add more SVG decals (so Designer can layer decals that need different styles[^1]).

## Related

- [TUP-1234](https://jira.tacc.utexas.edu/browse/TUP-1234)

## Changes

- **added** more SVG symbols i.e. more "TACC SVG Decal" Icon choices

## Testing & UI

1. Create two Icons:
    - `svg-cyber-clip-dots`
    - `svg-cyber-clip-stroke`
    - `svg-cyber-clip-ol`
2. To `svg-cyber-clip-dots`, add fill.
3. To `svg-cyber-clip-stroke`, add stroke.
3. To `svg-cyber-clip-ol`, add stroke.
4. Position `svg-cyber-clip-`**`dots`** and `svg-cyber-clip-`**`stroke`** icons atop each other.
5. Verify that together they look like a paper clip with some filled dots and some open circles.

**Live**: https://dev.tup.tacc.utexas.edu/test/icons/ (designer may be using)

![Screen Shot 2023-03-21 at 12 43 03](https://user-images.githubusercontent.com/62723358/226697275-6f39cabb-d653-4dca-a3c7-6f5b1676d725.png)

https://user-images.githubusercontent.com/62723358/226700725-5adc0915-df82-40d9-af5f-e9d2c38a7357.mov

[^1]: Our current SVG decal implementation relies on each SVG being available in an external file as a symbol. With external SVG's, CSS can only be applied (I think) to all parts of a single symbol. Thus, to get multiple styles in one SVG, I either must inline the SVG (a change to implementation that I can explore another day), or I let the designer combine multiple SVGs (this PR allows a clunky solution to do that).
